### PR TITLE
MINOR: Fix Streams examples in documentation

### PR DESCRIPTION
**Pull request by vahidhashemian**
Performed minor cleanup and escaped `<` and `>` so code examples are shown correctly in the browser.

### CODE REVIEW

File: docs/streams.html [hunk 1]
```diff
@@ -279,8 +279,8 @@
 <pre>
     KStreamBuilder builder = new KStreamBuilder();
 
-    KStream<String, GenericRecord> source1 = builder.stream("topic1", "topic2");
-    KTable<String, GenericRecord> source2 = builder.table("topic3", "stateStoreName");
+    KStream&lt;String, GenericRecord&gt; source1 = builder.stream("topic1", "topic2");
```
Comment (mjsax) [inline docs/streams.html:282]: Why is this required? It is within `<pre>` environment. Same applies to other changes similar to this one.

Comment (vahidhashemian) [reply docs/streams.html:282]: @mjsax At least my browser (Safari on Mac) doesn't show the types between `<` and `>` ([here](http://kafka.apache.org/documentation#streams_dsl_source)). It would be great if you can check and confirm whether it shows fine on your side. Thanks.

Comment (vahidhashemian) [reply docs/streams.html:282]: It seems characters, including `<` and `>`, still need to be escaped inside `<pre>`.

Comment (mjsax) [reply docs/streams.html:282]: Chrome and Firefox Mac also do not show the generics... Just wondering if `<code>` tag (instead of `<pre>` tag) wouldn't be a better fix?

Comment (vahidhashemian) [reply docs/streams.html:282]: That didn't work when I tried it. With only `<code>` the code wraps (it ignores the EOL character), and it still doesn't show the generics. With both `<pre>` and `<code>` it still doesn't show the generics.

**Review by mjsax — APPROVED** [review]

**Comment by mjsax** [comment]
@guozhangwang Call for review.

**Comment by guozhangwang** [comment]
LGTM and merged to trunk.
